website/src/content/docs/docs/migrate-swagger/faq/pipeline.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Resolving Pipeline failures
+---
+
+This document explains how to mitigate pipeline failures in TypeSpec migration PRs.
```

Comment:
There is already a document in the spec repo for this, we should at the very least link it as it contains more info

Reply:
we might also want to reorganize a bit here as this is at the same level and after the resolving "Breaking change failues" and resolving lint diff failures

Reply:
> There is already a document in the spec repo for this, we should at the very least link it as it contains more info

Which document you are saying? The content in this document only applies to migration cases.